stresstests/src/test/java/org/neo4j/causalclustering/stresstests/CatchUpLoad.java

```diff
@@ -76,6 +80,11 @@ protected void doWork()
             try
             {
                 cluster.removeReadReplicaWithMemberId( newMemberId );
+                if ( ex == null && deleteStore )
+                {
+                    fs.deleteRecursively( readReplica.storeDir() );
+                }
+                deleteStore = !deleteStore;
```

Comment:
To delete the store half of the time, so that a complete store copy is forced. This is what showed the issue consistently.